cli.js

```diff
@@ -41,6 +42,18 @@ var conf = pkgConf.sync('ava', {
 	}
 });
 
+// check for valid babel config shortcuts (can be either "default" or "inherit")
+var isValidShortcut = ['default', 'inherit'].indexOf(conf.babel) >= 0;
+
+if (conf.babel.length > 0 && !isValidShortcut) {
```

Comment:
Presumably this will crash if `babel` is set to `null`. I'd prefer this to be more explicit:

```js
if (!conf.babel || (typeof conf.babel !== 'object' && conf.babel !== 'default' && conf.babel !== 'inherit')) {
```

Note that `conf.babel` should default to `default` so `!conf.babel` would only occur if it's explicitly set to a falsey value.